src/main/java/seedu/address/logic/parser/DukeCooksParser.java

```diff
@@ -121,6 +129,18 @@ public Command parseCommand(String userInput) throws ParseException {
         case DeletePageCommand.COMMAND_WORD:
             return new DeletePageCommandParser().parse(arguments);
 
+        case AddTaskCommand.COMMAND_WORD:
+             return new AddTaskCommandParser().parse(arguments);
+
+        case DeleteTaskCommand.COMMAND_WORD:
+            return new DeleteTaskCommandParser().parse(arguments);
+
+        case EditTaskCommand.COMMAND_WORD:
+            return new EditTaskCommandParser().parse(arguments);
+
+        case FindTaskCommand.COMMAND_WORD:
+             return new FindTaskCommandParser().parse(arguments);
```

Comment:
I can't seem to run the program if I this part is here. It says that it had duplicate labels.